### DISCUSSION TITLE
Infinite scroll(load more button) with good performance

### DISCUSSION THREAD

**Discussion by yaquawa** [livewire/livewire#1289] [4. General]
The solution to achieve the infinite scroll or "load more" can be as simple as:

1. loop the collection in view
2. when the "load more" button clicked, push the new collection's items to the old one

But this can cause performance issue very quickly.
Because the server sends back the whole items of the collection but not the newly added ones.
The payload gets larger and larger every time you trigger a new click event on the load more button.

Infinite scroll is very common for a modern web app, I can't find any solution for it. Any ideas?

What I would like to see is something like this:
```php
// in the livewire component view
@loadMoreButton // generates a load more button and setup all the php and js for infinite scroll
```

**Comment by Wulfheart** [comment]
Related issue: https://github.com/livewire/livewire/discussions/5363
